test/helpers/TaskListHelperSpec.scala

```diff
@@ -270,9 +270,59 @@ class TaskListHelperSpec extends ControllerMockFixtures with SpecBase with Scala
 
         val listOfPages = Seq(ReporterStatusPage, DisclosureStatusPage)
 
-        haveAllJourneysBeenCompleted(listOfPages, userAnswers, index, isInitialDisclosureMarketable = false) mustBe false
+        haveAllJourneysBeenCompleted(listOfPages, userAnswers, index, mockDisclosure.copy(firstInitialDisclosureMA = Some(false))) mustBe false
 
       }
+
+      "must return false if user is doing a REPLACEMENT of an NEW DISCLOSURE that IS MARKETABLE and " +
```

Comment:
A previous test condition is: "must be true if user is doing REPLACEMENT DISCLOSURE OF ADDITIONAL DISCLOSURE & THAT IS NOT MARKETABLE but original DAC6NEW is marketable and has NOT started HALLMARKS or ARRANGEMENT DETAILS journey"  - The ticket says: For **ANY** replacement of an additional journey, InitialDisclosureMA flag should be defaulted to false. Is it a conflicting condition?

Reply:
I believe so - there was a test condition that should have been removed way back when a change was made to how additional worked. I believe all the tests here to be correct now 😄